internal/cmd/users.go

```diff
@@ -246,7 +229,9 @@ func UpdateUser(name string, cmdOptions editUserCmdOptions) error {
 
 	if !isSelf {
 		// Todo otp: update term to temporary password (https://github.com/infrahq/infra/issues/1441)
-		fmt.Fprintf(os.Stderr, "  Updated one time password for user.\n")
+		fmt.Fprintf(os.Stderr, "  Updated password for %q.\n", name)
+	} else {
+		fmt.Fprintf(os.Stderr, "  Updated password.\n")
```

Comment:
In other PRs and above in this PR I notice you're using `cli.Output` for this kind of message, but here it's going to stderr.  What's the reason for that?

I don't have a strong preference for this, I think either could be fine as long as we are consistent.

Reply:
No reason for this one. Will update it to use `cli.Output` instead